_data-prepper/monitoring.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Monitoring
+nav_order: 12
+---
+
+# Monitoring Data Prepper with Metrics
+
+Monitoring metrics in Data Prepper is instrumented using [Micrometer](https://micrometer.io/). There are two types of metrics: JVM and system metrics, and plugin metrics. [Prometheus](https://prometheus.io/) is used as the default metrics backend.
+
+## JVM and system metrics
+
+JVM and system metrics are runtime metrics used to monitor Data Prepper instances. They include metrics for classloaders, memory, garbage collection, threads, and others. For details, [JVM and system metrics](https://micrometer.io/docs/ref/jvm) for details. 
+
+### Naming
+
+JVM and system metrics in Data Prepper follow pre-defined names in Micrometer, such as `jvm_classes_loaded`, and `jvm_memory_used`. 
+
+### Serving
+
+By default, metrics are served from the **/metrics/sys** endpoint on the Data Prepper server. The format is a text Prometheus scrape. This port can be used for any frontend which accepts Prometheus metrics, such as [Grafana](https://prometheus.io/docs/visualization/grafana/). The configuration can be updated to serve metrics to other registries like CloudWatch which does not require or host the endpoint, but publishes the metrics directly to CloudWatch.
+
+## Plugin metrics
+
+Each plugin (if necessary, each class within a plugin) is responsible for posting relevant metrics. Besides custom metrics introduced by specific plugins, `data-prepper-api` already introduced the following common metrics for plugin types:
```

Comment:
@oeyh Is there a reference to the Data Prepper API?

Reply:
@chenqi0805 Is there a reference to the Data Prepper API?

Reply:
@natebower What is the correct way to include this link as we have the code block tick marks around `data-prepper-api` ?

Reply:
@carolxob If I understand your question correctly, I would link the text instead of using tick marks (the [Data Prepper API](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-api)).